packages/jest-haste-map/src/index.js

```diff
@@ -39,6 +39,7 @@ type Options = {
   cacheDirectory?: string,
   console?: Console,
   extensions: Array<string>,
+  forceNativeFind?: boolean,
```

Comment:
I don't understand. This option should specifically not use the find binary but should use the fs APIs instead.